packages/world/mud.config.ts

```diff
@@ -310,6 +309,15 @@ export default defineWorld({
       },
       key: ["entityId"],
     },
+    ReverseFragmentPosition: {
```

Comment:
Doesnt this indirection increase gas?

Reply:
it does, but with the new changes it felt more natural like this and more consistent, as we usually get the forcefield and the fragment entities with `getForceField`. But yeah we could try abstracting it away and using less tables, but also felt weird because for all other entities we always map entity -> data, and coord -> entity.

Reply:
ooc what's the gas increase? If you compare gas report on main and this branch? If its not that much then yea for consistency can keep the extra table

Reply:
on it